PAWSUITests/DataManagementTests.swift

```diff
@@ -39,5 +42,16 @@ final class DataManagementTests: XCTestCase {
         let refreshedECGText = app.staticTexts["ECG Recording"]
         XCTAssertTrue(refreshedECGText.waitForExistence(timeout: 2))
         XCTAssertEqual(initialECGText.description, refreshedECGText.description)
+        
+        // Now return to the Health app, and add some more ECGs before capturing a screenshot (for App Store).
+        for _ in 0..<10 {
+            try self.exitAppAndOpenHealth(.electrocardiograms)
+        }
```

Comment:
Wonder if we need 10 ECGs? How many fit on the screen? Cloud we save some time by doing less?

Is there any way to detect if the test has been run by fastlane snapshot (e.g. an ENV variable or something) or an argument passed in that we could only run this in case we want to take screenshots?

Reply:
Yes, we can check whether the app is in snapshot mode by accessing `UserDefaults.standard.bool(forKey: "FASTLANE_SNAPSHOT")`, which I have now added to the code. Also, I think we only need five ECGs at most for filling up the screen on the largest devices, so I revised the number of iterations accordingly.